test/unit/env.js

```diff
@@ -0,0 +1,27 @@
+import jQuery from 'jquery';
+
+global.$ = global.jQuery = jQuery;
+require('bootstrap');
+global.welderApiHost = 'localhost';
+global.welderApiScheme = 'HTTP';
+
+// <mockLocalStorage>
+const localStorageMock = (() => {
```

Comment:
We need a global window to attach our mocked localStorage API to in the Jest environment.